gossip/src/cluster_info.rs

```diff
@@ -3082,6 +3082,7 @@ fn filter_on_shred_version(
                 // Allow contact-infos so that shred-versions are updated.
                 CrdsData::LegacyContactInfo(_) => true,
                 CrdsData::NodeInstance(_) => true,
+                CrdsData::ContactInfo(_) => true,
```

Comment:
Can this match arm go first so it is just below the comment re contact-infos?

Reply:
ya np